src/amo/components/Link.js

```diff
@@ -18,7 +19,12 @@ export class LinkBase extends React.Component {
   }
 
   render() {
-    const { base, children, to } = this.props;
+    const { base, children, href, to } = this.props;
+
+    if (typeof href === 'string' && href.startsWith('/')) {
+      const linkHref = path.join(base, href);
+      return <a {...this.props} href={linkHref}>{children}</a>;
```

Comment:
I don't think spreading `this.props` is correct here. I think you meant to allow custom props to be passed along, right? If so, an approach like this would be safer:
````js
const { base, children, href, to, ...customProps } = this.props;
// ...
return <a {...customProps} href={linkHref}>{children}</a>;
````